server/db_mysql.go

```diff
@@ -195,6 +196,39 @@ func (ms *MsqlCache) GetFixity(start string, end string, item string, status str
 	return results
 }
 
+func (ms *MsqlCache) PostFixity(item string) (int, error)  {
+        var err error
+        fixity_results := ms.GetFixity("*", "*", item, "scheduled")
+        if len(fixity_results) > 0 {
+                err = ms.DeleteFixity(fixity_results[0].Id)
+                if err != nil {
+                        return 500, err
+                }
+        }
+
+        err = ms.UpdateFixity(item, "scheduled", "")
+        if err != nil {
+                       return 500, err
+        }
+        return 200, nil
+}
+
+func (qc *MsqlCache) PutFixity(id string) ( int, error)  {
```

Comment:
Again, the name is an HTTP verb. HTTP verbs have no place in the database code.

Maybe change the UpdateFixity signature to also handle the situation of changing the scheduled time?

Reply:
OK